### PR TITLE
Update README.md to make clear about open source parts

### PR DESCRIPTION
**Pull request by TheReal-Flo**
This PR fixes issue #593

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,12 +56,20 @@ If you need help with these solutions, please go to the support channel of the Q
...
 - **questcraft.net is not maintained or owned by anyone on the QuestCraft team**; they could put malware on it anytime. Please do not download anything from or visit questcraft.net. **Our new website is here at [QuestCraft](https://questcraft.org/)**. 
 - Pico (and ONLY Pico) support is planned and is currently in development. Until then, kindly use **[Vivecraft PCVR](https://www.vivecraft.org/)**.
 - When installing mods that are not from Mod Manager on your Quest, you may face issues, ranging from unexpected behaviour to crashes! If you experience any, please **[file an issue](https://github.com/QuestCraftPlusPlus/QuestCraft/issues/new/choose)** or go to the **[Discord server](https://discord.gg/questcraft/)**!
 
+# This project is open source...
+...tho you don't find any code parts in this repo. You can find the code for each subproject in their respective repos:
+- **[Pojlib (Launcher library)](https://github.com/questcraftplusplus/pojlib)**
+- **[VivecraftMod (Minecraft VR Mod)](https://github.com/questcraftplusplus/vivecraftmod)**
+- **[VulkanMod (Renderer)](https://github.com/QuestCraftPlusPlus/VulkanMod),**
```
Comment (thejudge156) [inline README.md:68]: We don't actually use VKMod nor have any plans for it anymore. Please remove this line.

**Review by thejudge156 — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -56,12 +56,20 @@ If you need help with these solutions, please go to the support channel of the Q
...
 - All support questions should be asked inside of the **[QuestCraft Discord](discord.gg/questcraft)** for the best experience.
 - **questcraft.net is not maintained or owned by anyone on the QuestCraft team**; they could put malware on it anytime. Please do not download anything from or visit questcraft.net. **Our new website is here at [QuestCraft](https://questcraft.org/)**. 
 - Pico (and ONLY Pico) support is planned and is currently in development. Until then, kindly use **[Vivecraft PCVR](https://www.vivecraft.org/)**.
 - When installing mods that are not from Mod Manager on your Quest, you may face issues, ranging from unexpected behaviour to crashes! If you experience any, please **[file an issue](https://github.com/QuestCraftPlusPlus/QuestCraft/issues/new/choose)** or go to the **[Discord server](https://discord.gg/questcraft/)**!
 
+# This project is open source...
+...tho you don't find any code parts in this repo. You can find the code for each subproject in their respective repos:
```
Comment (thewiilover) [inline README.md:65]: dont use the word tho. this is a professional project. use professional langauge.

Comment (thewiilover) [inline README.md:65]: also change "you dont find any code parts" to "you wont find the source code in this repo"

File: README.md [hunk 3]
```diff
@@ -56,12 +56,20 @@ If you need help with these solutions, please go to the support channel of the Q
...
+- **[Pojlib (Launcher library)](https://github.com/questcraftplusplus/pojlib)**
+- **[VivecraftMod (Minecraft VR Mod)](https://github.com/questcraftplusplus/vivecraftmod)**
+- **[VulkanMod (Renderer)](https://github.com/QuestCraftPlusPlus/VulkanMod),**
+- **[Unity Env (Launcher Wrapper)](https://github.com/QuestCraftPlusPlus/QCXR-XR-Wrapper).** 
+
+
+
```
Comment (thewiilover) [inline README.md]: also im not sure if questcraft uses vulkanmod anymore. dont quote me on that lol

**Review by thewiilover — CHANGES_REQUESTED** [review]
see comments

File: README.md (hunk 3 above)
Comment (thewiilover) [reply README.md]: i was right lol (thanks judge :3)

Comment (thejudge156) [reply README.md]: Correct. I already pointed that out though.

File: README.md (hunk 2 above)
Comment (thewiilover) [reply README.md:65]: or change the ending from "... in this repo" to " ...on this repo"

Comment (thejudge156) [reply README.md:65]: Good suggestion.

Comment (thejudge156) [reply README.md:65]: This should be a given.

File: README.md [hunk 4]
```diff
@@ -56,12 +56,18 @@ If you need help with these solutions, please go to the support channel of the Q
...
 - Pico (and ONLY Pico) support is planned and is currently in development. Until then, kindly use **[Vivecraft PCVR](https://www.vivecraft.org/)**.
 - When installing mods that are not from Mod Manager on your Quest, you may face issues, ranging from unexpected behaviour to crashes! If you experience any, please **[file an issue](https://github.com/QuestCraftPlusPlus/QuestCraft/issues/new/choose)** or go to the **[Discord server](https://discord.gg/questcraft/)**!
 
+# This project is open source...
+...but you don't find source code in this repo. You can find the source code for each subproject in their respective repos:
+- **[Pojlib (Launcher library)](https://github.com/questcraftplusplus/pojlib)**
+- **[VivecraftMod (Minecraft VR Mod)](https://github.com/questcraftplusplus/vivecraftmod)**
+- **[Unity Env (Launcher Wrapper)](https://github.com/QuestCraftPlusPlus/QCXR-XR-Wrapper).** 
+
+
```
Comment (thejudge156) [inline README.md:70]: Remove this extra line, I'll merge it then.

**Review by thejudge156 — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -62,7 +62,7 @@ If you need help with these solutions, please go to the support channel of the Q
 - When installing mods that are not from Mod Manager on your Quest, you may face issues, ranging from unexpected behaviour to crashes! If you experience any, please **[file an issue](https://github.com/QuestCraftPlusPlus/QuestCraft/issues/new/choose)** or go to the **[Discord server](https://discord.gg/questcraft/)**!
 
 # This project is open source...
-...tho you don't find any code parts in this repo. You can find the code for each subproject in their respective repos:
+...but you don't find source code in this repo. You can find the source code for each subproject in their respective repos:
```
Comment (thewiilover) [inline README.md:65]: change to "but you wont find the source code in/on this repo. You can find the source code for each module/submodule in their respective repos: "

Comment (thewiilover) [reply README.md:65]: the one with two words are up to the devs to choose

**Review by thewiilover — CHANGES_REQUESTED** [review]

File: README.md (hunk 5 above)
Comment (thewiilover) [reply README.md:65]: up to judge on wiether or mnot these are needed

Comment (TheReal-Flo) [reply README.md:65]: I would stay with in, because a Repo is basically like a drawer where things are inside, not a shelf with things on top

**Pull request merged**